features/step_definitions/verification_process_steps.rb

```diff
@@ -121,6 +121,10 @@ def user(*traits)
   find(:xpath, IvlDocumentsPage.view_history_option).click
 end
 
+And(/^Individual clicks on verify$/) do
+  find('li', :text => "Verify").click
```

Comment:
great, now we just need to replace the stuff inside find with a page object model

Reply:
can you provide the reference code?